lib/client-assets.php

```diff
@@ -849,6 +849,11 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		wp_die( $post_to_edit->get_error_message() );
 	}
 
+	/**
+	 * Apply filter to modify WP_Post prior to sending it to Gutenberg.
+	 */
+	$post_to_edit = apply_filters( 'after_gutenberg_gets_post_to_edit', $post_to_edit );
```

Comment:
At the risk of reigniting a bikeshed, I'd suggest avoiding the codename `gutenberg` in the names of hooks. See also #4681.

Reply:
Does `after_block_editor_gets_post_to_edit` work?

Reply:
Replaced `gutenberg` with `block_editor`.